lib/Command/Configure.php

```diff
@@ -313,6 +314,29 @@ private function configureSms(InputInterface $input, OutputInterface $output) {
 				$providerConfig->setUser($username);
 				$providerConfig->setApiKey($apiKey);
 
+				break;
+			case 'plivo':
+				$config->setProvider($provider);
+				/** @var PlivoConfig $providerConfig */
+				$providerConfig = $config->getProvider()->getConfig();
+				
+				$authIdQuestion = new Question('Please enter your plivo authentication id (Auth ID): ');
+				$authId = $helper->ask($input, $output, $authIdQuestion);
+				
+				$authTokenQuestion = new Question('Please enter your plivo authentication token (Auth Token): ');
+				$authToken = $helper->ask($input, $output, $authTokenQuestion);
+				
+				$srcNumberQuestion = new Question('Please enter your plivo phone number (in E.164 format): ');
+				$srcNumber = $helper->ask($input, $output, $srcNumberQuestion);
+				
+				$callbackUrlQuestion = new Question('Please enter your plivo callback url: ');
+				$callbackUrl = $helper->ask($input, $output, $callbackUrlQuestion);
+				
+				$providerConfig->setValue($providerConfig::AUTH_ID_KEY,$authId);
+				$providerConfig->setValue($providerConfig::AUTH_TOKEN_KEY, $authToken);
+				$providerConfig->setValue($providerConfig::CALLBACK_URL, $srcNumber);
+				$providerConfig->setValue($providerConfig::SRC_NUMBER_KEY, $callbackUrl);
```

Comment:
You switched something up here ;)

Reply:
Yes, I noticed other improvements I can make as well. I plan to submit those changes in another PR.

Reply:
In addition, I see the mistake I made, thanks.